content/ko/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -70,7 +70,7 @@ <h2 style="color: #3771e3;">스케일링 개요</h2>
                             <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling1.svg">
                         </div>
 
-                        <div class="item">
+                        <div class="item ">
```

Comment:
@mhamdisemah , Thank you for your contribution.
However, the change for the content/**ko**/docs/tutorials/kubernetes-basics/scale/scale-intro.html seems a mistake.
In the preview (https://deploy-preview-15179--kubernetes-io-master-staging.netlify.com/ko/docs/tutorials/kubernetes-basics/scale/scale-intro/) ,
Carousel is not working for the Korean document.

Reply:
this PR is for ZH document not for KO

Reply:
@mhamdisemah :wave: This PR contains Korean content change which causes a conflict. Please resolve the conflict and remove the "language/ko" label.